xtask/src/cargo.rs

```diff
@@ -48,6 +48,25 @@ impl Package {
     /// Update the version of this crate in dependant crates' manifests, with the given version
     /// prefix.
     pub(crate) fn update_dependants(&self, metadata: &Metadata, dry_run: bool) -> Result<()> {
+        let workspace_manifest_path = metadata.workspace_root.join("Cargo.toml");
+        let mut document = read_file(&workspace_manifest_path)?.parse::<Document>()?;
+        let workspace_deps = &mut document["workspace"]["dependencies"];
+
+        if workspace_deps.get(&self.name).is_some() {
```

Comment:
Aren't all ruma crates in the workspace dependencies? Shouldn't we return an error or even panic if the dependency is not found here?

Reply:
Ruma itself isn't, but I guess we can add more checks regardless.